content/2021-07-19-brig-init.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Brig Init"
+slug: "2021-07-19-brig-init"
+description: "A deep dive into one of our favorite new features"
+date: "2021-07-19"
+author: "Kent Rancourt & Willie Yao"
+---
+# Brig Init
+
+Last week, we announced the first beta release of Brigade 2. Today, we're taking a deep dive into an exciting new feature we shipped in that release: the Brigade CLI's new `init` command.
+
+Brigade 2 development efforts have been heavily focused on improving Brigade's UX. A significant pain point from Brigade 1.x that we've been intent on improving was the interactive, but onerous project creation process. Since the project's maintainers are big fans of "infrastructure as code" and because imitation is the sincerest form of flattery, Brigade 2 drew inspiration from Kubernetes and utilizes project definition files that are reminiscent of Kubernetes manifests (but aren't). Projects can be created and updated by submitting such definitions to Brigade.
+
+While project definitions as source offer an attractive improvement to Brigade, they aren't without their own all-new frustration. _Just how does one author one of these project definitions?_ We don't know about you, but most of us on the Brigade team can't write a Kubernetes manifest from scratch! We generally copy an existing one that's close to what we need and then modify it. But, we felt we could do better for Brigade 2's project definitions.
```

Comment:
optional nit/style: perhaps `However, we felt we could do better for Brigade 2's project definitions.`